practica-2023/main.js

```diff
@@ -143,12 +167,21 @@ function setButtonEvents(eventID){
   const decreaseButton = document.querySelector('#decrementButton-' + eventID);
   const quantityInput = document.querySelector('.input-' + eventID);
   const checkoutButton = document.querySelector('#checkoutButton-' + eventID);
+  const selectTickets = document.querySelector('.select-' + eventID);
 
   increaseButton.addEventListener("click", (event) => {
     var value = parseInt(quantityInput.value);
```

Comment:
Try as much as possible not to use `var` in JS :). 
Either user `let` or `const` which are block-scoped while `var` is not. 
This means it can be modified outside the scope of the function which is usually not the intended behavior.

Reply:
Understood. Removed all unintended var uses :)